pkg/scraper/bruneida_scraper.go

```diff
@@ -0,0 +1,82 @@
+package scraper
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/b-open/jobbuzz/pkg/types"
+	"github.com/gocolly/colly"
+)
+
+func ScrapeBruneida() types.Jobs {
+	allowedDomain := colly.AllowedDomains("www.bruneida.com")
+	linkCollector := colly.NewCollector(
+		allowedDomain,
+	)
+	jobCollector := linkCollector.Clone()
+
+	jobs := types.Jobs{}
+
+	// Scraping the links
+	linkCollector.OnHTML("ul.list-az.ul-azs", func(e *colly.HTMLElement) {
+		e.ForEach(".az-detail>h3.az-title>a.h-elips", func(i int, child *colly.HTMLElement) {
+			link := child.Attr("href")
+			jobCollector.Visit(link)
+
+		})
+	})
+
+	// Scraping the jobs
+	jobCollector.OnHTML("body", func(h *colly.HTMLElement) {
+		job_title := h.ChildText("#title-box-inner div.inline-block.pull-left h1")
+		company := h.ChildText("#ad-contact ul li:first-child span.bb b.small")
+		salary := h.ChildText("#ad-body-inner .opt .opt-dl:nth-child(3) .dd")
+
+		location := ""
+		h.ForEach("#ad-body-inner .opt .opt-dl", func(i int, child *colly.HTMLElement) {
+			title := child.ChildText(".dt")
+
+			if strings.Contains(title, "City") || strings.Contains(title, "Local") {
+				s := child.ChildText(".dd")
+
+				if s == "" {
+					return
+				}
+
+				location += s + " "
+			}
+		})
+
+		location = strings.TrimSuffix(location, " ")
```

Comment:
Are you trying to join a bunch of string together?

I think something like this might be better?

```go
locations := []string{}
h.ForEach("#ad-body-inner .opt .opt-dl", func(i int, child *colly.HTMLElement) {
			title := child.ChildText(".dt")

			if strings.Contains(title, "City") || strings.Contains(title, "Local") {
				locations = append(locations, child.ChildText(".dd"))
			}
		})
location := strings.Join(locations, " ")
```